Moving Heidi's and Camila's personas to contributors file (#1310)
alan-turing-institute/the-turing-way: book/website/afterword/contributors-record.md

> This is my motivation to translate the book to Spanish, and I hope that the Spanish version will be used as an important resource on the master course we are developing in  LA-CoNGA physics project.

* More information

> I’m from Venezuela, and although I have done most of my career in Europe I’ve been always keen to stay connected to the academic and scientific wold back in Latin-America.
> I’m the co-founder of the CEVALE2VE project (http://www.cevale2ve.org/en/home/), which of a virtual learning community that aims to tackle the serious issue of brain-drain in some Latin-American countries by bringing back the knowledge in a digital/online platform.
> More recently that project has become consolidated into LA-CoNGA physics (http://laconga.redclara.net/), an EU Erasmus+ funded project with a mission to create a Latin American and European Community for Advanced Physics.
> In this project I’m helping to build a data science module that will be thought in an online master course.

### Heidi Seibold

* GitHub id: [HeidiSeibold](https://github.com/HeidiSeibold)
* ORCID id: [0000-0002-8960-9642](https://orcid.org/0000-0002-8960-9642)
* Short bio:

> > I am a medical AI researcher, research software engineer and open science expert at LMU Munich, Bielefeld University and the Helmholtz Zentrum Munich.

Reviewer: ```suggestion
> I lead a group on Open AI in Health at the Helmholtz Zentrum Munich.
```
